source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -247,6 +247,14 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
               metricItem["json_Collections"] = metricCollections.to_json
               metricItems.push(metricItem)
 
+              if metricValue > 10000000000
+                telemetryProperties = {}
+                telemetryProperties["metricJSON"] = JSON.generate(metricJSON)
```

Comment:
This will be huge (in the order of MBs) if the node has 100 or more containers, right? do we need all the payload?

Reply:
removed